File: app/src/main/java/com/karthik/splash/Models/PhotoDetail/PhotoDetailInfo.java
package com.karthik.splash.Models.PhotoDetail;

import java.util.List;
import com.google.gson.annotations.SerializedName;


public class PhotoDetailInfo {

	@SerializedName("current_user_collections")
	public List<CurrentUserCollectionsItem> currentUserCollections;

	@SerializedName("color")
	public String color;

	@SerializedName("created_at")
	public String createdAt;

	@SerializedName("description")
	public String description;

	@SerializedName("liked_by_user")
	public boolean likedByUser;

	@SerializedName("urls")
	public Urls urls;

	@SerializedName("updated_at")
	public String updatedAt;

	@SerializedName("downloads")
	public int downloads;

	@SerializedName("width")
	public int width;

	@SerializedName("location")
	public Location location;

	@SerializedName("links")
	public Links links;

	@SerializedName("id")
	public String id;

	@SerializedName("categories")
	public List<CategoriesItem> categories;

	@SerializedName("user")
	public User user;

	@SerializedName("height")
	public int height;

	@SerializedName("likes")
	public int likes;

	@SerializedName("exif")
	public Exif exif;
}

File: app/src/main/java/com/karthik/splash/models/photodetail/PhotoDetailInfo.kt
package com.karthik.splash.models.photodetail

import com.google.gson.annotations.SerializedName


data class PhotoDetailInfo(
        @SerializedName("color")
        var color: String? = null,
        @SerializedName("created_at")
        var createdAt: String? = null,
        @SerializedName("description")
        var description: String? = null,
        @SerializedName("liked_by_user")
        var likedByUser: Boolean = false,
        @SerializedName("updated_at")
        var updatedAt: String? = null,
        @SerializedName("downloads")
        var downloads: Int = 0,
        @SerializedName("width")
        var width: Int = 0,
        @SerializedName("location")
        var location: Location? = null,
        @SerializedName("id")
        var id: String? = null,
        @SerializedName("user")
        var user: User? = null,
        @SerializedName("height")
        var height: Int = 0,
        @SerializedName("likes")
        var likes: Int = 0
)
